openmls/TEST-DESIGN.md

```diff
@@ -0,0 +1,175 @@
+# OpenMLS testing proposition
+
+The purpose of this document is to improve tests in openmls and try to agree on a common standard for writing them.
+
+## Current status
+
+Here are some factual remarks about the current state of tests.
+
+1. erratic: no clear pattern emerges. It looks like tests have been piled up at different times by different persons.
```

Comment:
That's how testing works, everyone adds tests for their code.

Reply:
I've precised that one